lib/cpu.js

```diff
@@ -0,0 +1,91 @@
+var os = require('os')
+var fs = require('fs')
+var exec = require('child_process').exec
+
+/**
+ * Gathers Clock, PageSize and system uptime through /proc/uptime
+ * This method is mocked in procfile tests
+ */
+function updateCpu (cpu, next) {
+  if (cpu !== null) {
+    getRealUptime(function (err, uptime) {
+      if (err) return next(err)
+      cpu.uptime = uptime
+      next(null, cpu)
+    })
+    return
+  }
+
+  getClockAndPageSize(function (err, data) {
+    if (err) return next(err)
+
+    getRealUptime(function (err, uptime) {
+      if (err) return next(err)
+
+      cpu = {
+        clockTick: data.clockTick,
+        pageSize: data.pageSize,
+        uptime: uptime
+      }
+
+      next(null, cpu)
+    })
+  })
+}
+
+module.exports = updateCpu
```

Comment:
It's a good practice to keep this line at the end of the file.
Maybe you can move also updateCpu at the bottom near the export to make the code more readable.
But it's quite personal keep that just as a suggestion.

Reply:
I prefer to put them straight after the exported function. Like this if a function is not public you know it when you read the code. Ideally it could even be before the function itself. Anyway no particular rule here according to standardjs linter let's keep it that way.